readme.md

```diff
@@ -162,15 +186,25 @@ Your Rails app has the following `application.html.erb`. Nothing shows up in you
 </html>
 ```
 
-> Your answer...
+<h1><%= link_to "Quiz", root_path %></h1>
+  <nav>
+    <% if current_user %>
+      <%= link_to "Create New User, new_user_path %>
+      <%= link_to "Log Out #{current_user.email}", destroy_user_session_path, method: :delete %>
+    <% else %>
+      <%= link_to "Sign Up", new_user_registration_path %>
+      <%= link_to "Sign In", new_user_session_path %>
+    <% end %
+
+You would add this code into to get your application.html.erb to show up on your browser.
 
 ## Question 13
 
 Of the three options below, which is the most "correct" way of organizing the files that make up an Angular app, as used in class? Why is this option considered "better" than the other two?
 
-> Your answer...
+The most correct way is A, it is considered better because it is more organized with the parent folders.
```

Comment:
[Take a look at the solution.](https://github.com/ga-dc/wdi7_quiz10/tree/solution#question-13)